modules/caddyhttp/fileserver/staticfiles.go

```diff
@@ -246,6 +246,20 @@ func (fsrv *FileServer) Provision(ctx caddy.Context) error {
 		fsrv.precompressors[ae] = p
 	}
 
+	// check sort options
+	for idx, sortOption := range fsrv.SortOptions {
+		switch idx {
+		case 0:
```

Comment:
This works; but we should also verify the length of SortOptions is 1 or 2 if not empty.